scalekit/client.py

```diff
@@ -204,65 +207,137 @@ def generate_client_token(self, client_id: str, client_secret: str) -> str:
             response = json.loads(response.content)
             return response
         except Exception as exp:
-            raise exp
+            from scalekit.common.exceptions import ScalekitException
+            raise ScalekitException(str(exp))
```

Comment:
Need not raise caught exceptions as Scalekit Exceptions. We only need to raise our custom exceptions (Arising out of our validations) as ScalekitException()

```suggestion
            raise exp
```